src/lib/mutect.ml

```diff
@@ -2,9 +2,27 @@ open Common
 open Run_environment
 open Workflow_utilities
 
+module Configuration = struct
+
+  type t = { name: string; parameters: (string * string) list; }
+
+  let to_json {name; parameters}: Yojson.Basic.json =
+    `Assoc [
+      "name", `String name;
+      "parameters",
+      `Assoc (List.map parameters ~f:(fun (a, b) -> a, `String b));
+    ]
+  let render {parameters; _} =
+    List.concat_map parameters ~f:(fun (a,b) -> [a; b])
+
+  let default =
+    {name = "default"; parameters = []}
+
+end
+
 let run ?(reference_build=`B37)
     ~(run_with:Machine.t) ~normal ~tumor ~result_prefix
-    ?(more_edges = []) how =
+    ?(more_edges = []) ~configuration how =
```

Comment:
He's put the optional-with-default at the `Pipeline.t` level → https://github.com/hammerlab/biokepi/pull/93/files#diff-6cc6e1c3d49a4d36b4884380ad53d585R177

One day we should make everything more homogeneous :)